src/mame/misc/magic10.cpp

```diff
@@ -115,6 +143,16 @@ Both setups show different variants for components layout, memory size, NVRAM, e
   - Priorities,likely to be hardwired with the color writes (0=tile has the
      highest priority).
   - Define parent/clone relationship between Magic's 10 and Music Sort.
+  - Position of "PUSH IP START" and "SUPER PETRIX" logo don't match actual hardware
+    but fixing these causes the poker winplan to be in the wrong place.
+  - Super Petrix and Super Gran Safari use the same board but spetrix accesses the sound
+    chip on the upper byte of the data bus and sgsafari on the lower. Is this just
+	a programming error on spetrix ?
```

Comment:
Remember that the 68000 has “byte smearing” – when writing a byte, the value appears on both halves of the data bus.  If the chip select for the sound chip ignores the lane select signals, it doesn’t matter whether you write to the odd or even byte of the word, it will work either way.

Other examples where this happens include the Macintosh Display Card, where firmware will write to odd or even offsets and expect it to work because the card ignores lane select, and Sega Mega Drive software that expects byte writes to the I/O blocks (controller and expansion ports) to work on odd or even offsets.

You should probably use the same address map for both, and use trampolines like `m68k_ioport_data_read` and `m68k_data_write` in `md_base_state` (src/mame/sega/megadriv.cpp) to deal with it.